packages/maskbook/src/plugins/Snapshot/apis/index.ts

```diff
@@ -66,7 +66,7 @@ export async function vote(identifier: ProposalIdentifier, choice: number, addre
         },
     })
 
-    const sig = await Services.Ethereum.sign(msg, address)
+    const sig = await Services.Ethereum.personalSign(msg, address)
```

Comment:
cc @guanbinrui

Reply:
I found that `sign()` does not awake metamask sign interface window well.